tool/tsh/common/device.go

```diff
@@ -147,3 +158,26 @@ func (c *deviceKeygetCommand) run(cf *CLIConf) error {
 	fmt.Printf("DeviceCredential %s\n", val)
 	return nil
 }
+
+type deviceActivateCredentialCommand struct {
+	*kingpin.CmdClause
+	encryptedCredential       string
+	encryptedCredentialSecret string
+}
+
+func (c *deviceActivateCredentialCommand) run(cf *CLIConf) error {
+	err := dtnative.HandleTPMActivateCredential(
+		c.encryptedCredential, c.encryptedCredentialSecret,
+	)
+	if cf.Debug {
+		if err != nil {
+			// On error, wait for user input before executing. This is because this
+			// opens in a second window. If we return the error immediately, then
+			// this window closes before the user can inspect it.
+			log.WithError(err).Error("An error occurred during credential activation. Press enter to close this window.")
+			_, _ = fmt.Scanln()
+			return trace.Wrap(err)
+		}
+	}
+	return trace.Wrap(err)
```

Comment:
Simplify?

```suggestion
	if cf.Debug && err != nil {
		// On error, wait for user input before executing. This is because this
		// opens in a second window. If we return the error immediately, then
		// this window closes before the user can inspect it.
		log.WithError(err).Error("An error occurred during credential activation. Press enter to close this window.")
		_, _ = fmt.Scanln()
	}
	return trace.Wrap(err)
```

Reply:
819c6596af